frontend/src/App.tsx

```diff
@@ -1,13 +1,15 @@
-import { useMemo } from 'react';
-import AppRouter from './Router';
-import { AppContext } from './contexts';
-import getSupabaseClient from './utils/supabase';
+import { useMemo } from "react";
```

Comment:
Is there a chance you can change your prettier settings to prefer single quotes over double quotes?
It's temporary fix until we set up prettier for the project properly